### PR TITLE
Added instructions to remove KubeFlow again

### PR DESCRIPTION
**Pull request by rummens**
Added useful instructions on how to remove KubeFlow, I happen to use this if I broke something in KF and want to reset it. It was not in the documentation but I can imagine it helps someone.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/464)

### CODE REVIEW

File: content/docs/started/getting-started.md [hunk 1]
```diff
@@ -78,6 +78,14 @@ Download, set up, and deploy (If you prefer to work from source code, feel free
 **Important**: The commands above will enable collection of **anonymous** user data to help us improve Kubeflow; for more information including instructions for explicitly
 disabling it please refer to the [usage reporting guide](/docs/other-guides/usage-reporting/).
 
+## Remove KubeFlow
```
Comment (sarahmaddox) [inline content/docs/started/getting-started.md]: s/KubeFlow/Kubeflow

File: content/docs/started/getting-started.md [hunk 2]
```diff
@@ -78,6 +78,14 @@ Download, set up, and deploy (If you prefer to work from source code, feel free
...
+To remove KubeFlow use the same script as above. Note, that it will delete the namespace kubeflow along with everything you have deployed in it!
```
Comment (sarahmaddox) [inline content/docs/started/getting-started.md]: Recommended wording to improve clarity:

```
To remove your Kubeflow deployment, you can use the same `kfkctl` script as above.
```

Comment (rummens) [reply content/docs/started/getting-started.md]: Sure sounds nice as well. Lets go with that. How can I change it?

**Comment by sarahmaddox** [comment]
Hallo @rummens How's it going with the updates to this PR? To make updates, you can just change the files and push them to the same branch (rummens:patch-2). That will update the files in this PR.

**Comment by rummens** [comment]
> Hallo @rummens How's it going with the updates to this PR? To make updates, you can just change the files and push them to the same branch (rummens:patch-2). That will update the files in this PR.

Done, thanks for the tipp :-)

**Pull request merged**